src/plugin.js

```diff
@@ -1,97 +1,49 @@
 /* eslint-disable no-restricted-syntax, no-shadow */
 
-const fs = require('fs');
-const path = require('path');
-const fetch = require('node-fetch');
 const parseCssUrls = require('css-url-parser');
+const { helpers } = require('@eik/common');
+const fetch = require('node-fetch');
 
 const notUrl = (url) => url.substr(0, 4) !== 'http';
 
-const notBare = (str) =>
-    str.startsWith('/') || str.startsWith('./') || str.startsWith('../');
-
-async function readJSONFile(path) {
-    try {
-        const contents = await fs.promises.readFile(path);
-        return JSON.parse(contents);
-    } catch (err) {
-        return {};
-    }
-}
-
-async function readEikJSONMaps(eikJSONPath, pkgJSONPath) {
-    const eikJSON = await readJSONFile(eikJSONPath);
-    const pkgJSON = await readJSONFile(pkgJSONPath);
-
-    if (eikJSON.name && pkgJSON.eik) {
-        throw new Error('Eik configuration was defined in both in package.json and eik.json. You must specify one or the other.');
-    }
-
-    const config = { ...eikJSON, ...pkgJSON.eik };
-
-    if (typeof config['import-map'] === 'string') return [config['import-map']];
-    return config['import-map'] || [];
-}
-
 async function fetchImportMaps(urls = []) {
```

Comment:
Looks like this code is copy pasted between the different repos. Wondering if it would make sense to push this into the common package so that the error handling can be kept consistent over time.

Reply:
Yes. There is a bit of copied code between the plugins which can be DRYed up into `@eik/common`. The whole `.fetchImportMaps()` is a candidate too. Though, the other plugins need a little bit of love before I want to do that.